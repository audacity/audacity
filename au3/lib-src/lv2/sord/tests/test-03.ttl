# Test ; operator
@prefix : <http://example.org/base#> .
:a :b :c ;
   :d :e ;
   :f :g .
