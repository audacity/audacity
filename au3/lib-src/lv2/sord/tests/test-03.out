<http://example.org/base#a> <http://example.org/base#b> <http://example.org/base#c> .
<http://example.org/base#a> <http://example.org/base#d> <http://example.org/base#e> .
<http://example.org/base#a> <http://example.org/base#f> <http://example.org/base#g> .
